lib/perfdata/influxdbwriter.cpp

```diff
@@ -290,6 +290,10 @@ void InfluxdbWriter::CheckResultHandlerWQ(const Checkable::Ptr& checkable, const
 
 			Dictionary::Ptr fields = new Dictionary();
 			fields->Set("value", pdv->GetValue());
+			fields->Set("unit", pdv->GetUnit());
```

Comment:
fields->Set("unit", pdv->GetUnit()); is already added (https://github.com/Icinga/icinga2/blob/1f0c6bbdc1d619802467723ad73e1619055434c5/lib/perfdata/influxdbwriter.cpp#L309)